infra/install_bootstrap_dependencies.sh

```diff
@@ -8,8 +8,12 @@ umask 022
 echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
 curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
 
+# For gcloud
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+
 sudo apt update
-sudo apt install -y docker.io python3-pip openjdk-8-jdk-headless jq skopeo
+sudo apt install -y docker.io python3-pip openjdk-8-jdk-headless jq skopeo google-cloud-cli google-cloud-cli-gke-gcloud-auth-plugin
 sudo snap install --classic kubectl
```

Comment:
What is snap and why don't we install kubectl using apt? https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#apt Oddly, that page thinks the auth plugin should be called `google-cloud-sdk-gke-gcloud-auth-plugin`, note the sdk in place of cli.

Reply:
hm weird, I'll double check this by creating a bootstrap vm again and get gcloud + kubectl properly installed.

Reply:
Still need to double check this.